.github/workflows/ci.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     - name: Check Lake
       run: make check-lake
     - name: Test Lake
-      run: make test-ci -j4
+      run: make test-ci
```

Comment:
I assume you changed this to make the CI output more readable (at the cost of speed)?

Reply:
Yes, it was very hard to see which test failed and what its output was.  (Which is particularly important on platforms that you can't easily test on except for CI.)  The tests are still faster than each of the two builds we do in CI.